client/src/components/Navbar.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+// import {
+//   FormControlLabel,
+//   Switch,
+//   Badge,
+//   Avatar,
+//   styled,
+// } from '@mui/material';
+// import MailIcon from '@mui/icons-material/Mail';
+// import NotificationsIcon from '@mui/icons-material/Notifications';
+import { StyledNav } from '../styles/Stylesheet';
+// import defaultUserIcon from '../img/default-user-icon.png';
+
+// const StyledBadge = styled(Badge)(({ theme }) => ({
+//   '& .MuiBadge-badge': {
+//     backgroundColor: '#44b700',
+//     color: '#44b700',
+//     boxShadow: `0 0 0 2px ${theme.palette.background.paper}`,
+//     '&::after': {
+//       position: 'absolute',
+//       top: 0,
+//       left: 0,
+//       width: '100%',
+//       height: '100%',
+//       borderRadius: '50%',
+//       animation: 'ripple 1.2s infinite ease-in-out',
+//       border: '1px solid currentColor',
+//       content: '""',
```

Comment:
everything look great try to minimize those commented  lines of code on your next PR!!